client/src/app/containers/Game/CribHand.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import Grid from '@material-ui/core/Grid';
+import GridList from '@material-ui/core/GridList';
+
+import PlayingCard from './PlayingCard';
+
+const CribHand = props => {
+  if (!props.cards) {
+    return null;
+  }
```

Comment:
:cactus: this works; the way I've seen it done more often in react is
``jsx
return !props.cards && (<>rest of jsx here</>)
```

Reply:
:til: